### PR TITLE
Unify into a single Helm chart

### PR DESCRIPTION
**Pull request by lkrcal**

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -1,24 +1,90 @@
...
+
+      pool:
+        vmImage: 'Ubuntu-16.04'
+
+      steps:
+        - task: HelmDeploy@0
+          displayName: 'Helm init'
+          inputs:
+            connectionType: Kubernetes Service Connection
+            kubernetesServiceEndpoint: 'gke_focal-freedom-236620_asia-southeast1_lkrcal-6809978de35e5bad'
```
Comment (rushminatorr) [inline azure-pipelines.yml]: Do you want to have a conversation on deploying this to an existing cluster deployed by our automation - but on a different namespace? And we could use gcloud to query for cluster endpoints and kubeconfig.

File: iofog/templates/connector.yml [hunk 2]
```diff
@@ -15,6 +15,19 @@ spec:
       labels:
         name: connector
     spec:
+      initContainers:
+        - name: wait-for-controller
+          image: fmiguelez/alpine-curl-jq:latest
```
Comment (rushminatorr) [inline iofog/templates/connector.yml:20]: which one better - curl or kubectl for querying for IP?

File: iofog/templates/setup.yml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+          - name: CONTROLLER_URL
+            value: "http://{{ .Values.controller.host }}:{{ .Values.controller.port }}/api/v3"
+        command:
+          - /bin/sh
+          - -c
+          - |
+            echo -n "Creating user John Doe... "
+            RET=$(curl --request POST --url "${CONTROLLER_URL}/user/signup" \
+                --header 'Content-Type: application/json' \
+                --data '{"email":"user@domain.com","firstName":"John","lastName":"Doe","password":"#Bugs4Fun"}' \
```
Comment (rushminatorr) [inline iofog/templates/setup.yml:36]: Just a comment - for future, we could pass some secret file that can be stored as secret and not be open like this.

File: iofog/templates/connector.yml (hunk 2 above)
Comment (lkrcal) [reply iofog/templates/connector.yml:20]: This is just to wait for the status to be online, the IP does not need to be retrieved here.

File: azure-pipelines.yml (hunk 1 above)
Comment (lkrcal) [reply azure-pipelines.yml]: Done. Gcloud gets credentials (kubeconfig), Helm does the rest.

**Pull request merged**